packages/lexical-rich-text/src/index.ts

```diff
@@ -388,11 +390,15 @@ function onPasteForRichText(
   });
 }
 
-function onCopyForRichText(event: ClipboardEvent, editor: LexicalEditor): void {
+function onCopyForRichText(
+  event: CommandPayloadType<typeof COPY_COMMAND>,
```

Comment:
Could we just use a simple union type here instead? It might be a little more readable and `CommandPayloadType` feels slightly too general if we're not using it a lot elsewhere. 

```suggestion
  event: ClipboardEvent | KeyboardEvent,
```

Reply:
@thegreatercurve I actually started with just a union type. The question then becomes, do we want to provide a utility type for extracting the payload in a nice way from the `lexical` package, to make it a bit easier for other users if they want to extract a function for delegating to from the command handlers? Inside of the command handlers the payload is inferred, but that's only the case if/when it is inlined. If you want to do something like this:

```ts
type MY_COMMAND = createCommand<string | SomeOtherType>();

// ...
editor.registerCommand(MY_COMMAND, myCommandHandler);

// ...
function myCommandHandler(payload: string | SomeOtherType) {}
```

The idea here was to provide functionality for this, such that the payload type and the argument type are can easily be kept in sync if that union type would grow for some reason

```ts
function myCommandHandler(payload: CommandPayloadType<typeof MY_COMMAND>) {}
```

If we do want to provide it, it would make a lot of sense to adorn it with a proper TSDoc comment to show its purpose and some examples of how to use it.

I guess it's also a question of:
* How often do the generic type of a command really change?
* Where would we want type errors in usage to pop up?